### ISSUE TITLE
CardTokenPaymentMethod class is assigned to PaymentMethod attribute in OrderRequest

### ISSUE THREAD

**Issue by lmeyer** [hipay/hipay-fullservice-sdk-php#58]
Hi, 

The [documentation](https://developer.hipay.com/online-payments/sdk-reference/sdk-php) tells us to do the following :

```php
$paymentMethod = new \HiPay\Fullservice\Gateway\Request\PaymentMethod\CardTokenPaymentMethod();   
$paymentMethod->cardtoken = "61f92d7a135db52dbd583b2aad208e73978196392357f674bacf39f549042f14";
$paymentMethod->eci = 7;
$paymentMethod->authentication_indicator = 0;
 
$orderRequest->paymentMethod = $paymentMethod;
```

But I get an error from psalm, which seems to be right :

$orderRequest->paymentMethod with declared type `HiPay\Fullservice\Gateway\Model\PaymentMethod` cannot be assigned type `HiPay\Fullservice\Gateway\Request\PaymentMethod\CardTokenPaymentMethod`

Can we avoid this error ? Maybe it's a bug ?

Regards,

**Comment by legithubdeaymeric** [comment]
@lmeyer 

Thanks for your report, we take a look asap.

**Comment by VincentLanglet** [comment]
Could be solved by https://github.com/hipay/hipay-fullservice-sdk-php/pull/61

**Comment by JBaptisteLvt** [comment]
Hello !
This has been fixed in version 2.11.4, released on February 27th 2023
